review.go

```diff
@@ -15,15 +15,40 @@ const (
 	ReviewReasonRule            ReviewReasonType = "rule"
 )
 
+// ReviewParams is the set of parameters that can be used when approving a review.
+type ReviewParams struct {
+	Params `form:"*"`
+}
+
+// ReviewApproveParams is the set of parameters that can be used when approving a review.
+type ReviewApproveParams struct {
+	Params `form:"*"`
+}
+
+// ReviewListParams is the set of parameters that can be used when listing reviews.
+type ReviewListParams struct {
+	ListParams   `form:"*"`
+	Created      *int64            `form:"created"`
+	CreatedRange *RangeQueryParams `form:"created"`
+}
+
 // Review is the resource representing a Radar review.
 // For more details see https://stripe.com/docs/api#reviews.
 type Review struct {
-	Charge   *Charge          `json:"charge"`
-	Created  int64            `json:"created"`
-	ID       string           `json:"id"`
-	Livemode bool             `json:"livemode"`
-	Open     bool             `json:"open"`
-	Reason   ReviewReasonType `json:"reason"`
+	Review        *Review          `json:"review"`
```

Comment:
I think this is supposed to be `Charge`/`*Charge`/`json:"charge"` right?

Reply:
Damn nice catch, I messed up my string replace